src/modules/dashboard/lists/BorrowAssetsList/BorrowAssetsListItem.tsx

```diff
@@ -62,7 +63,36 @@ export const BorrowAssetsListItem = ({
         value={Number(variableBorrowRate)}
         incentives={vIncentivesData}
         symbol={symbol}
-      />
+      >
+        {symbol === 'ETH' && (
```

Comment:
I think you also need to check that the market is v3 ethereum because the rewards will be only there
https://governance.aave.com/t/arfc-merit-a-new-aave-alignment-user-reward-system/16646#budget-and-scope-10